packages/estatico-copy/test/index.js

```diff
@@ -16,18 +16,28 @@ const defaults = {
   },
 };
 
-test.cb('default', (t) => {
-  task(defaults)().on('finish', () => utils.compareFiles(t, path.join(__dirname, 'expected/default/*')));
-});
+test.afterEach(() => del(path.join(__dirname, '/results')));
+
+test('default', t => new Promise((resolve) => {
+  task(defaults)()
```

Comment:
can we re-write this to something less condensed? this is kinda hard to read.

```js
const result = task(defaults);
result().on('finish', () => { ... });
```

would work, too, right?